### ISSUE TITLE
Documentation?

### ISSUE THREAD

**Issue by AnastasiaLS** [jexp/jetbrains-heroku-plugin#32]
I just downloaded your plugin - it seems to have installed just fine, API key went in just fine. However, I cannot seem to figure out how to actually use your plugin in Ruby Mine for deployment. The URL for the download (http://plugins.jetbrains.com/plugin/6659?pr=phpStorm) gives a full documentation link to the Wiki here, but the page is empty. I dug through the folders here on GitHub, and didn't find a README anywhere.

Is there documentation somewhere that provides instructions for use? If so, would it be possible to have them uploaded here?  Thanks!

**Comment by James9074** [comment]
Did you ever figure this one out?

**Comment by rice-big0923** [comment]
I am in the same boat. No idea how to deploy my app.

**Comment by jexp** [comment]
Hi sorry,

you have to enable vcs integration for git.

And then there is a connect icon on the left (the socket on the top) which allows you to connect your app to your heroku app.

You can also create new heroku apps for your local application.

You're right I should add some docs and perhaps do a screencast.

**Comment by AnastasiaLS** [comment]
Yeah, I fiddled around and figured it out. But yes, some documentation - or a walk-through video - would be awesome to add.

**Comment by rice-big0923** [comment]
Yes, that would be very helpful. I finally gave up and pushed to Heroku manually via the terminal. But I will keep trying to do it through Rubymine.

**Comment by francescobovoli** [comment]
@jexp , I am feeling very stupid, I enabled vcs integration for git (and saved the whole repository on github), but I still can't find the connection icon you mention:
"And then there is a connect icon on the left (the socket on the top) which allows you to connect your app to your heroku app."

For the records, by using Help -> Find Action, I searched for heroku and found a "Heroku" entry with a purple icon. Clicking it adds a new panel at the bottom called "Heroku" (after showing me a error popup with no error message in it). The panel is completely empty. Screenshot included for reference.

BTW, happy to include this in the wiki if you help me out :)
![image](https://cloud.githubusercontent.com/assets/6606649/5165347/3c0a1556-73db-11e4-8a60-a4e72ada44d7.png)

**Comment by michaelminter** [comment]
> And then there is a connect icon on the left (the socket on the top) which allows you to connect your app to your heroku app.

Is this step missing in RubyMine 7+?

**Comment by YodasMyDad** [comment]
I'm guessing this doesn't work anymore then?

**Comment by James9074** [comment]
Yeah I never was able to get it to work...

**Comment by Beengie** [comment]
I struggled, but found it
Preferences> Build, Execution, Deployment > Clouds
Then Heroku was automatically in there ready to configure.

Hope that helps
